migrate-dao.js

```diff
@@ -1,7 +1,22 @@
 const utils = require('./utils.js')
 const sanitize = require('./sanitize')
+const path = require('path')
+const fs = require('fs')
+let stateFile = path.join(__dirname, 'deployment-state.json')
+
+function writeState (state) {
+  fs.writeFileSync(stateFile, JSON.stringify(state, undefined, 2), 'utf-8')
+}
 
 async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logTx, previousMigration, customabislocation }) {
+  let deploymentState
+
+  if (fs.existsSync(stateFile)) {
+    deploymentState = JSON.parse(fs.readFileSync(stateFile))
```

Comment:
I guess that this file need to be deleted after successful deployment .
also , there should be a special cmd something like `migrate -restart` which ignore or delete this file

Reply:
Yes it is deleted at the end. I will add a `-restart` flag

Reply:
Added